pocs/state/states/default/pointing.py

```diff
@@ -59,8 +62,8 @@ def on_enter(event_data):
                     'Waiting for images: {} seconds'.format(wait_time))
                 pocs.status()
 
-                if wait_interval > timeout:
-                    raise error.Timeout
+                if wait_time > timeout:
```

Comment:
I've been writing a bunch of similar timeout code. I think we probably need a utility so that we get this right all the time. PySerial contains a timeout class.

Reply:
Yes, I was going to mention the exact same thing. This particular code isn't dealing with pyserial though and it's already a separate process. Could be made more generic and useful for sure.